build/buildconstants/params_calibnet.go

```diff
@@ -102,8 +102,8 @@ const UpgradeWaffleHeight = 1779094
 // 2024-10-23T13:30:00Z
 const UpgradeTuktukHeight = 2078794
 
-// ??????
-const UpgradeXxHeight = 999999999999999
+// 2024-12-16T23:00:00Z
+const UpgradeTeepHeight = 2235454
```

Comment:
This seems right given my estimation, but I wasn't sure a calibration calculator I could use to confirm.

Reply:
I now have this in my bash profile:

```sh
filecoin_now() {
        (date +%s; echo 1598306400 - 30 / p) | dc
}

filecoin_time() {
        date -Iseconds -d "@"$((echo $1; echo '30 * 1598306400 + p') | dc)
}

filecoin_time_utc() {
        date -u -Iseconds -d "@"$((echo $1; echo '30 * 1598306400 + p') | dc)
}

calibnet_now() {
        (date +%s; echo 1667326380 - 30 / p) | dc
}

calibnet_time() {
        date -Iseconds -d "@"$((echo $1; echo '30 * 1667326380 + p') | dc)
}

calibnet_time_utc() {
        date -u -Iseconds -d "@"$((echo $1; echo '30 * 1667326380 + p') | dc)
}
```

```
$ calibnet_time_utc 2235454
2024-12-16T23:00:00+00:00
```